cmd/prometheus/main.go

```diff
@@ -376,7 +377,7 @@ func main() {
 	a.Flag("web.enable-remote-write-receiver", "Enable API endpoint accepting remote write requests.").
 		Default("false").BoolVar(&cfg.web.EnableRemoteWriteReceiver)
 
-	supportedRemoteWriteProtoMsgs := config.RemoteWriteProtoMsgs{config.RemoteWriteProtoMsgV1, config.RemoteWriteProtoMsgV2}
+	supportedRemoteWriteProtoMsgs := remoteapi.MessageTypes{remoteapi.WriteV1MessageType, remoteapi.WriteV2MessageType}
```

Comment:
We could replace config entries as well with remoteapi perhaps? 🤔

Reply:
Ok so I did most of the constants, but some still remain. I'll modify that in next PR for replacing client/sending code.